server/src/main/java/org/elasticsearch/script/Metadata.java

```diff
@@ -39,6 +39,7 @@
  */
 public class Metadata {
     protected static final String INDEX = "_index";
+    protected static final String REDIRECT = "_redirect";
```

Comment:
I'm unsure if `_redirect` should also be added to `IndexRequest`. I guess that could potentially expose `_redirect` to the `_bulk` API where this doesn't make much sense. It's also more like an alias to `_index` but with slightly different semantics.

@dakrone do you have a suggestion?